package.json

```diff
@@ -20,14 +20,15 @@
   "dependencies": {
     "axios": "^0.18.0",
     "body-parser": "^1.18.3",
-    "cassandra-driver": "^3.5.0",
     "chance": "^1.0.16",
     "css-loader": "^0.28.11",
     "express": "^4.16.3",
     "faker": "^4.1.0",
     "jquery": "^3.3.1",
     "mongoose": "^5.1.3",
+    "morgan": "^1.9.0",
```

Comment:
Doesn't look like 'morgan' is used? If it is not, then I'd erase this to avoid confusion later down the road.